### DISCUSSION TITLE
How can I enable fallback access point?

### DISCUSSION THREAD

**Discussion by testpaul999** [syssi/esphome-jk-bms#475] [Q&A]
Hi,
How can I enable fallback access point?
have try 
ap:
 ssid: fallback 
 password: xyz

unable to compile.
Thx for any hints.
Regards

**Comment by syssi** [comment]
Could you provide the error message? Did you try a clean build?

**Comment by testpaul999** [comment]
![image](https://github.com/syssi/esphome-jk-bms/assets/35597907/b15ae31a-e50d-4162-8aff-bb31cbe473e3)

![Uploading image.jpg…]()

**Comment by testpaul999** [comment]
Have found a solution
WiFi and ap together won’t work.
Regards

**Reply by syssi** [reply]
The station mode and the accesspoint mode as fallback should work in general. You YAML code looks wrong.
